client/src/components/ContentNavigator/ContentDataProvider.ts

```diff
@@ -723,11 +723,6 @@ class ContentDataProvider
           icon = "folder";
           break;
       }
-    } else {
-      const extension = item.name.split(".").pop().toLowerCase();
```

Comment:
Am I understanding correctly? By removing this case, we return undefined, and `package.json` basically dictates the icon for `.sas` extension files, right? (looking at https://github.com/sassoftware/vscode-sas-extension/blob/95794c63f4cd8ffa23f3dd14583a71f1f867ce77/package.json#L1110-L1113)

If this is accurate, this LGTM

Reply:
yes correct